src/main/java/com/fidesmo/tutorials/hellofidesmo/MainActivity.java

```diff
@@ -100,42 +102,40 @@ protected void onNewIntent(Intent intent) {
 
     @Override
     public void tagDiscovered(Tag tag) {
-        Log.i(TAG, "Card detected on the NFC interface!");
+        setMainMessage("Reading card...");
```

Comment:
My reasoning was: "why are we using R.string.xxx for all the other messages but not here"? Looks not very coherent. The reason for using it in the first place was to apply best practices even for such a little tutorial, as a way of saying that we care for quality.